### ISSUE TITLE
Add New Tag?

### ISSUE THREAD

**Issue by robmoggach** [bennylope/django-taggit-labels#10]
Would be great if this also included a way to create a new tag...

Perhaps the last tag in the list is a "plus" symbol and when you click on it, it becomes a text field styled like an unselected tag...

**Comment by robmoggach** [comment]
Looks like this jquery plugin could be useful - would need to create an ajax view possibly for creating new tags... https://maxfavilli.com/jquery-tag-manager

**Comment by morenoh149** [comment]
I may have to implement this for my project. @bennylope any direction here before I submit a PR?

**Comment by bennylope** [comment]
Nothing specifically @morenoh149 - as long as the necessary JS requirements are bundled I'd say go for it.

**Comment by djjudas21** [comment]
@morenoh149 did you ever make any progress on this? :smile:

**Comment by morenoh149** [comment]
Nope. But I may do this as part of a youtube video? Want to help?

**Comment by djjudas21** [comment]
I'm happy to help if I can, although I'm a Python noob and I have zero experience of js/jquery/etc

**Comment by morenoh149** [comment]
ok @djjudas21 please email me. We can setup sometime to discuss how to do this. I am very experienced in js.
